lib/cli/ui/spinner/spin_group.rb

```diff
@@ -76,6 +76,7 @@ def pause_spinners(&block)
         def initialize(auto_debrief: true, interrupt_debrief: false, max_concurrent: 0, work_queue: nil)
           @m = Mutex.new
           @tasks = []
+          @messages = []
```

Comment:
should it be called `above_messages` to be more clear where they appear?

Reply:
this is internal to the object and we don't have messages that get printed elsewhere, but if it feels unclear then yeah i can rename it

Reply:
up to you

Reply:
changed it to `@puts_above`